sap-lama-apps/azuredeploy.json

```diff
@@ -0,0 +1,622 @@
+            }
+        },
+        "location": {
+          "type": "string",
+          "defaultValue": "[resourceGroup().location]",
+          "metadata": {
+            "description": "Location for all resources."
+          }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/sap-lama-apps/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "images": {
+            "Windows Server 2012 Datacenter": {
+                "sku": "2012-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "Windows Server 2012 R2 Datacenter": {
+                "sku": "2012-R2-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "Windows Server 2016 Datacenter": {
+                "sku": "2016-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "SLES 12": {
+                "sku": "12-SP3",
+                "offer": "SLES-SAP",
+                "publisher": "SUSE",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            },
+            "RHEL 7": {
+                "sku": "7.3",
+                "offer": "RHEL",
+                "publisher": "RedHat",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            },
+            "Oracle Linux 7": {
+                "sku": "7.3",
+                "offer": "Oracle-Linux",
+                "publisher": "Oracle",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            }
+        },
+        "internalOSType": "[variables('images')[parameters('osType')].OSType]",
+        "osConfigs": {
+            "Windows": {
+                "Publisher": "Microsoft.Compute",
+                "Name": "CustomScriptExtension",
+                "Version": "1.7",
```

Comment:
set autoUpgradeMinorVersion to true

Reply:
The VM extensions that use these variables have "autoUpgradeMinorVersion": true,